engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetInputTextureFromFBO.java

```diff
@@ -114,20 +112,13 @@ public StateChange getDefaultInstance() {
     }
 
     private int fetchTextureId() {
-
         // TODO: make checks to verify the FBOs has the requested buffer;
-
         if (inputFbo != null) {
-
             switch (textureType) {
-
                 case ColorTexture:
                     return inputFbo.colorBufferTextureId;
 
-                case DepthRenderBuffer:
-                    return inputFbo.depthStencilRboId;
-
-                case DepthTexture:
+                case DepthStencilTexture:
```

Comment:
I'll add the comment as you suggest for now.

However, and I *might* be wrong here, I don't think it's particularly required, mainly because the enum is FboTextureTypes, and not FboAttachmentTypes, and depthStencilRboId is definitely not the former.